diff --git a/node_modules/@types/splitting/index.d.ts b/node_modules/@types/splitting/index.d.ts
index 1cac9cb..501119b 100644
--- a/node_modules/@types/splitting/index.d.ts
+++ b/node_modules/@types/splitting/index.d.ts
@@ -26,7 +26,7 @@ export as namespace Splitting;
  * ```
  * @see {@link <https://splitting.js.org/guide.html#basic-usage>}
  */
-declare function Splitting(options?: Splitting.Options): Splitting.Result;
+declare function Splitting(options?: Splitting.Options): Splitting.Result[];
 
 declare namespace Splitting {
     /**
